Acornfile

```diff
@@ -10,11 +10,16 @@ args: {
 
 	// The tag that should be displayed in version information
 	tag: "v0.0.0-dev"
+
+	// The hostname of the database
+	dbHost: "test"
```

Comment:
To keep with the previous behavior, I'd make this whatever the local value is going to be.

Reply:
Updated to be `db`, which should default to using the container.